File: src/api/dotnet/FiniteDomainExpr.cs
/*++
Copyright (<c>) 2012 Microsoft Corporation

Module Name:

    FiniteDomainExpr.cs

Abstract:

    Z3 Managed API: Finite-domain Expressions

Author:

    Christoph Wintersteiger (cwinter) 2015-12-02

Notes:

--*/
using System.Diagnostics;
using System;

namespace Microsoft.Z3
{
    /// <summary>
    /// Finite-domain expressions
    /// </summary>
    public class FiniteDomainExpr : Expr
    {
        #region Internal
        /// <summary> Constructor for DatatypeExpr </summary>
        internal FiniteDomainExpr(Context ctx, IntPtr obj)
            : base(ctx, obj)
        {
            Debug.Assert(ctx != null);
        }
        #endregion
    }
}

File: src/api/java/FiniteDomainExpr.java
/**
Copyright (c) 2012-2014 Microsoft Corporation
   
Module Name:

    FiniteDomainExpr.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2015-12-02

Notes:
    
**/ 

package com.microsoft.z3;

/**
 * Finite-domain expressions
 **/
public class FiniteDomainExpr<R> extends Expr<FiniteDomainSort<R>>
{
    /**
     * Constructor for FiniteDomainExpr
     * @throws Z3Exception on error
     **/
    FiniteDomainExpr(Context ctx, long obj)
    {
        super(ctx, obj);
    }
}
